packages/kmod-6.1-nvidia-r570/nvidia-tesla-tmpfiles.conf

```diff
@@ -1,4 +1,7 @@
 C /etc/drivers/nvidia-tesla.toml
 C /etc/drivers/nvidia-open-gpu.toml
 C /etc/drivers/nvidia-open-gpu-copy-only.toml
+C /etc/drivers/nvidia-grid.toml
+C /etc/drivers/nvidia-grid-copy-only.toml
+C /etc/containerd/nvidia.env - - - - /usr/share/factory/nvidia/tesla/nvidia-path.env
```

Comment:
Why did this file come back?

Reply:
Messy rebase took an old file, I'll remove it.